Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+fossaCred = [
+    $class: 'StringBinding',
+    credentialsId: 'fossa-api-key',
+    variable: 'FOSSA_API_KEY'
+]
+
+hubCred = [
+    $class: 'UsernamePasswordMultiBinding',
+    usernameVariable: 'REGISTRY_USERNAME',
+    passwordVariable: 'REGISTRY_PASSWORD',
+    // credentialsId: 'coreeng-hub.docker.com',
+    credentialsId: 'orcaeng-hub.docker.com',
+]
+
+githubCred = [
+    $class: 'StringBinding',
+    variable: 'GITHUB_AUTH_TOKEN',
+    credentialsId: '3b0aa268-7549-4a0f-8cf4-8d75383bd05f', // 5K API rate limit
+]
+
+@Library('jps')
+_
+
+pipeline{
+  agent {
+    node {
+      label 'ubuntu-1804-overlay2'
+    }
+  }
+  triggers {
+    pollSCM('H * * * *')
+  }
+  environment{
+    FOSSA_API_KEY=credentials('fossa-api-key')
+    REGISTRY=credentials('orcaeng-hub.docker.com')
+  }
+  options{
+    disableConcurrentBuilds()
+  }
+  stages{
+    stage("FOSSA Analyze"){
+      steps {
+
+        withGithubStatus('FOSSA.scan'){
+
+          sh script: 'docker login -u $REGISTRY_USR -p $REGISTRY_PSW'
```

Comment:
You might not need this anymore since we have a public `fossa-analyzer` docker image